### DISCUSSION TITLE
App Hub Meeting - April 12, 2021

### DISCUSSION THREAD

**Discussion by amcgee** [dhis2/notes#255] [Meetings] [answered]
When: April 12, 2021 @ 11:00 CET
Where: Remote

## Agenda:
- [Review Submitted Apps](https://docs.google.com/spreadsheets/d/1gdIyBM3e5zk9LtsqfEARyI59PVU2UfUvpaCJvLPkwMc/edit#gid=0)
  - [OMS / WHO relationship tracing app](https://apps.dhis2.org/user/app/474731e4-cd9b-4f7d-9075-6b1ee4c64ee8)?
  - Start review of legacy "approved" apps
- App Hub improvements
  - (devops) AWS deployment pipeline
  - (devops) error analytics, uptime monitoring?
  - (devops) process for database backups, manual psql
  - Source Code Links mandatory?
  - Blog about new [security recommendations](https://developers.dhis2.org/docs/guides/apphub-guidelines#security)?
  - [Users migration](https://jira.dhis2.org/projects/HUB/issues/HUB-105)
  - (security) enforce app-meta match on upload (manifest version number, slug, id, developer, etc MUST MATCH what's in the App Hub DB)
  - We really should have App Hub email integrations (i.e. [HUB-17](https://jira.dhis2.org/projects/HUB/issues/HUB-17?filter=allopenissues)), should we spec this out?
- Continuous delivery
  - [Publish script](https://github.com/dhis2/app-platform/pull/532) ready to go?
  - Plan for application versioning, min/max versions on App Hub
  - Plan for core app uploads (team effort?)

**Comment by amcgee** [comment]
CC @Birkbjo @mediremi @deboragaleano @varl @cooper-joe @scottrusspatrick @larshelge @Philip-Larsen-Donnelly

**Comment by amcgee** [comment, marked as answer]
Notes here #259

**Reply by varl** [reply]
Final: https://github.com/dhis2/notes/blob/master/decisions/2021/04/12-app-hub-meeting.md
